backend/src/v2/cmd/launcher-v2/main.go

```diff
@@ -28,23 +27,26 @@ import (
-	componentSpecJSON = flag.String("component_spec", "", "The JSON-encoded ComponentSpec.")
-	importerSpecJSON  = flag.String("importer_spec", "", "The JSON-encoded ImporterSpec.")
-	taskSpecJSON      = flag.String("task_spec", "", "The JSON-encoded TaskSpec.")
-	podName           = flag.String("pod_name", "", "Kubernetes Pod name.")
-	podUID            = flag.String("pod_uid", "", "Kubernetes Pod UID.")
-	mlmdServerAddress = flag.String("mlmd_server_address", "", "The MLMD gRPC server address.")
-	mlmdServerPort    = flag.String("mlmd_server_port", "8080", "The MLMD gRPC server port.")
-	logLevel          = flag.String("log_level", "1", "The verbosity level to log.")
-	publishLogs       = flag.String("publish_logs", "true", "Whether to publish component logs to the object store")
-	cacheDisabledFlag = flag.Bool("cache_disabled", false, "Disable cache globally.")
+	copy                        = flag.String("copy", "", "copy this binary to specified destination path")
+	pipelineName                = flag.String("pipeline_name", "", "pipeline context name")
+	runID                       = flag.String("run_id", "", "pipeline run uid")
+	parentDagID                 = flag.Int64("parent_dag_id", 0, "parent DAG execution ID")
+	executorType                = flag.String("executor_type", "container", "The type of the ExecutorSpec")
+	executionID                 = flag.Int64("execution_id", 0, "Execution ID of this task.")
+	executorInputJSON           = flag.String("executor_input", "", "The JSON-encoded ExecutorInput.")
+	componentSpecJSON           = flag.String("component_spec", "", "The JSON-encoded ComponentSpec.")
+	importerSpecJSON            = flag.String("importer_spec", "", "The JSON-encoded ImporterSpec.")
+	taskSpecJSON                = flag.String("task_spec", "", "The JSON-encoded TaskSpec.")
+	podName                     = flag.String("pod_name", "", "Kubernetes Pod name.")
+	podUID                      = flag.String("pod_uid", "", "Kubernetes Pod UID.")
+	mlmdServerAddress           = flag.String("mlmd_server_address", "", "The MLMD gRPC server address.")
+	mlmdServerPort              = flag.String("mlmd_server_port", "8080", "The MLMD gRPC server port.")
+	logLevel                    = flag.String("log_level", "1", "The verbosity level to log.")
+	publishLogs                 = flag.String("publish_logs", "true", "Whether to publish component logs to the object store")
+	cacheDisabledFlag           = flag.Bool("cache_disabled", false, "Disable cache globally.")
+	mlPipelineServiceTLSEnabled = flag.Bool("ml_pipeline_service_tls_enabled", false, "Set to true if mlpipeline api server serves over TLS (default: false).")
+	metadataServiceTLSEnabled   = flag.Bool("metadata_tls_enabled", false, "Set to true if metadata server serves over TLS (default: false).")
+	caCertPath                  = flag.String("ca_cert_path", "", "The path to the CA certificate.")
```

Comment:
This might be need else where too and I may have missed it.
```suggestion
	caCertPath                  = flag.String("ca_cert_path", "", "The path to the CA certificate to trust on connections to the ML pipeline API server and metadata server.")
```